src/App.js

```diff
@@ -1,18 +1,27 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import { Route, Switch, NavLink, Redirect } from 'react-router-dom';
+import Home from './containers/Home';
+import Favorites from './containers/Favorites';
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+        <ul>
+          <li>
+            <NavLink to="/">Trending gifs</NavLink>
+          </li>
+          <li>
+            <NavLink to="/favorites">Mis favoritos</NavLink>
+          </li>
+        </ul>
+
+        <Switch>
+          <Route exact path="/" component={Home} />
+          <Route exact path="/favorites" component={Favorites} />
+          <Redirect to="/" />
```

Comment:
Remember you can pass an object to know where did the user try to access or something and catch that on the component on the redirect, it is always nice to let the user know what happened before the redirect